src/Display.js

```diff
@@ -17,18 +17,29 @@ class Display extends React.Component {
   }
 
   componentDidMount = async () => {
+    this.getMovies();
+  }
+
+  getMovies = async () => {
+    console.log('getting movies');
     const getMovies = await axios.post(`${API_SERVER}/user`, {email: this.props.auth0.user.email})
```

Comment:
this is going to need to be a get request right?

Reply:
I think this route needs to be changed to /movies too, it wont be /user

Reply:
updated to .get

Reply:
updated route to /movies